### DISCUSSION TITLE
Dc Market

### DISCUSSION THREAD

**Discussion by Estrella2019** [Crowns-of-Power/CoP#18] [Ideas]
We are working on fixing the Dc market they all sell the same things more or less so I was wondering what are some ideas you have to add to them? I was thinking of more items for remnants and adding more items for gold maybe even the colossal pots? What do you have in mind?

**Comment by TehNo0b** [comment]
What sort of scope?  Adding old existing things, or new things?  Limited to existing datablocks or?  Is talking about, suggesting, and planning **new content hush hush**?

**Reply by Estrella2019** [reply]
new and old any ideas would be nice the market is so out of date and ppl only use it for the f2p bags for silver

**Reply by TehNo0b** [reply]
DC vendors are all pretty unique in the goods they sell until you get to the ones by the Arena, those are all useless and mostly just there for looks.  I would remove all of those, or rework them significantly as it's a waste to even have them as they are.  Changing one to a token vendor, one to a fragment vendor, and one just being more unique with what you can purchase with gold would be a good idea.  I tried to keep current token vendors sort of organized like existing vendors ie: one sells gear, one sells utilities, one sells lesser things for fragments but the utility one could be broken up further as it has alot of things.. maybe too many and this gives you a start on atleast one of the new vendors.. utilities, potions etc.  I would steer clear of gold but it's my personal opinion.  That potion you wanted to sell for gold, I would sell a stack for tokens (or fragments) instead.  I'm alittle biased so you can take that into consideration.. I'm stuck on Gold = Bad!

Now the whole point of the token system we've put in place was meant to be an alternate path to obtain things.. maybe not the best things, but things that are a good alternative if you can't get that hard to get item in X raid zone cause it's just so rare.  While I like things being rare (that makes them unique!) I don't like dead ends on progress.  So I would propose you create a legit token set for people who can't get to all these raids,  People who want to carve their own path in this game and don't want to get a ton of handouts, or have to pay points bartering in a trade... time can and should be able to get everything if your willing to invest it.  I would align this set somewhere along with DT (for now).. not as good as DT perhaps but comparable atleast.. maybe even debatable if it's better because it focuses on (exampe) Myst instead of Intel, or melee crit instead of strength or whatever.  To make these items unique I suggest that they not need to be gemmed, in the datablock you can set it to not be gemmable.. and the stats you give it should reflect this.  I said make them comparable to DT.. so for example we'd probably basically be making them more like 3 Gemmed DT gear rather than 4 gemmed.. or somewhere inbetween.

Their are lots of things that you could/should put in the token system but this is pretty much where you should start now that my initial tests have been running for over 10 months now.  Eventually the goal should be that you could even release really rare hard to get NOTRADE ROYAL GEAR but you have some nice irresistable token sinks prior to getting there.  You have to build up the system to reach your desired destination.  It should ALMOST rival existing drop system, always get updated but always be just one tiny step behind. 

I could go on and on but no one seems to be able to follow along for very long so I'm gonna just stop here for now...
